frontend/src/services/jokes.js

```diff
@@ -0,0 +1,90 @@
+const getPage = async (numJokesPerPage, page) => {
+    const response = await fetch(`/joke/?num_jokes_per_page=${numJokesPerPage}&page=${page}`);
+
+    if (!response.ok) {
+        const message = `An error has occured: ${response.status}`;
+        throw new Error(message);
+    }
+
+    const body = await response.json();
+    console.log(body)
+    return body;
+};
+
+const get = async (id) => {
+    const response = await fetch(`/joke/${id}`);
+
+    if (!response.ok) {
+        const message = `An error has occured: ${response.status}`;
+        throw new Error(message);
+    }
+
+    const joke = await response.json();
+    return joke;
+};
+
+const del = async (id) => {
+    const settings = {
+        method: 'DELETE',
+    }
+
+    const response = await fetch(`/joke/${id}`, settings);
+
+    if (!response.ok) {
+        const message = `An error has occured: ${response.status}`;
+        throw new Error(message);
+    }
+
+    const success = await response.ok;
+    return success;
```

Comment:
.ok is not an awaitable thing, its a boolean property in the response object (which you already wait for)
```suggestion
    return response.ok;
```